runtime/lua/vim/lsp/util.lua

```diff
@@ -866,11 +867,21 @@ function M.convert_input_to_markdown_lines(input, contents)
     if input.kind then
       local value = input.value or ''
       list_extend(contents, split_lines(value))
+      -- Check that the content contains blank lines and is wrapped by codeblock
```

Comment:
This feels like an extremely specific fix. Why not just sent contents to `{}` if it consists of a single empty line? Having the length check feels a bit too much.